_posts/2025-07-31-security-releases.md

```diff
@@ -0,0 +1,46 @@
+---
+title: July 2025 Security Releases
+description: Security releases for Multer and On-headers has been published. We recommend that all users upgrade as soon as possible.
+tags: security vulnerabilities
+authors:
+  - name: Ulises Gascón
+    github: UlisesGascon
+---
+
+The Express team has released a new path version of [Multer](https://www.npmjs.com/package/multer) addressing a high-severity security vulnerability. Also a new minor version of [on-headers](https://www.npmjs.com/package/on-headers) addressing a low-severity security vulnerability
```

Comment:
```suggestion
The Express team has released a new patch version of [Multer](https://www.npmjs.com/package/multer) addressing a high-severity security vulnerability, and a new minor version of [on-headers](https://www.npmjs.com/package/on-headers) addressing a low-severity security vulnerability.
```

Reply:
Solved in [`4ee2d89` (#1994)](https://github.com/expressjs/expressjs.com/pull/1994/commits/4ee2d89c485ead8b7e75b91767367bde5d41e0d8)